.github/workflows/trivy.yml

```diff
@@ -0,0 +1,43 @@
+name: Container Security
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  schedule:
+    - cron: '29 19 * * 4'
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    permissions:
+      contents: read 
+      security-events: write 
+      actions: read
+    name: Build
+    runs-on: "ubuntu-20.04"
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 #v4.2.1
+
+      - name: Build an image from Dockerfile
+        run: |
+          docker build -t docker.io/my-organization/my-app:${{ github.sha }} .
```

Comment:
why not using a image that get's properly built instead of building it again?

Reply:
@derrix060 The main reason for building is to ensure transparency and to make sure the image is always built from the latest code. But this is also an option.

Reply:
I think we could hook this step in the pipeline where we build the images, so before publishing the image we check that it's okay, what do you think?